networking/hardware_networks/using-dpdk-and-rdma.adoc

```diff
@@ -28,6 +39,13 @@ include::modules/nw-sriov-rdma-example-mellanox.adoc[tag=content]
 [id="additional-resources_using-dpdk-and-rdma"]
 == Additional resources
 
+* xref:../../scalability_and_performance/cnf-create-performance-profiles.adoc#cnf-about-the-profile-creator-tool_cnf-create-performance-profiles[Creating a performance profile].
+* xref:../../scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc#adjusting-nic-queues-with-the-performance-profile_cnf-master[Reducing NIC queues using the Node Tuning Operator].
+* xref:../../scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc#performance-addon-operator-provisioning-worker-with-real-time-capabilities_cnf-master[Provisioning a worker with real-time capabilities].
+* xref:../../networking/hardware_networks/installing-sriov-operator.adoc#installing-sr-iov-operator_installing-sriov-operator[Installing the SR-IOV Network Operator]
+* xref:../../networking/hardware_networks/configuring-sriov-device.adoc#nw-sriov-networknodepolicy-object_configuring-sriov-device[Configuring an SR-IOV network device]
+* xref:../../networking/multiple_networks/configuring-additional-network.adoc#nw-multus-whereabouts_configuring-additional-network[Dynamic IP address assignment configuration with Whereabouts]
+* xref:../../scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc#disabling_interrupt_processing_for_individual_pods_cnf-master[Disabling interrupt processing for individual pods]
```

Comment:
This xref also needs updating after a rebase, due to the renamed page. The updated xref is:

````
* xref:../../scalability_and_performance/cnf-low-latency-tuning.adoc#disabling_interrupt_processing_for_individual_pods_cnf-master[Disabling interrupt processing for individual pods]
````